core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -486,6 +486,12 @@ interface Debug {
     @WithDefault("false")
     boolean enableDashboardDump();
 
+    /**
+     * Include a reasons entries in the generated json configuration files.
+     */
+    @WithDefault("false")
```

Comment:
I believe this will be `true` when Mandrel 24.1 is out?

Reply:
Not necessarily when Mandrel 24.1 is out. It will make sense to toggle it when Quarkus switches to the new `reachability-metadata.json` configuration though.